spec/css-spec.coffee

```diff
@@ -20,6 +20,19 @@ describe 'CSS grammar', ->
       expect(tokens[8]).toEqual value: ';', scopes: ['source.css', 'meta.property-list.css', 'meta.property-value.css', 'punctuation.terminator.rule.css']
       expect(tokens[10]).toEqual value: '}', scopes: ['source.css', 'meta.property-list.css', 'punctuation.section.property-list.end.css']
 
+    # Needs more complex examples
```

Comment:
While looking through the css specs on MDN, I vaguely remember (consider that this is a while ago) seeing things that simply weren't specced in language-css. I have looked into them, but for some reason (remember, this is a while ago) that led to rewriting a bigger part of the grammar which I deemed a bit out of the scope of a simple fix for an issue.